samples/net/virtual/src/main.c

```diff
@@ -44,7 +44,7 @@ struct virtual_test_context {
 static void virtual_test_iface_init(struct net_if *iface)
```

Comment:
The changes are good. Minor nit, the commit message of `samples/net/virtual: Increase sprintk buffers to hold parameter types` is copied from other commits and it talks about AT command strings but that is not the case for this commit (the string is not describing AT command).

Reply:
oops. cut&paste commit messages. Fixed.